package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "loopbench",
-  "version": "1.2.0",
+  "version": "1.2.1",
```

```suggestion
  "version": "1.2.0",
```

I'll bump this on release